src/Skia/Avalonia.Skia/FramebufferRenderTarget.cs

```diff
@@ -78,5 +78,11 @@ public IDrawingContextImpl CreateDrawingContext(IVisualBrushRenderer visualBrush
             canvas.ResetMatrix();
             return new DrawingContextImpl(canvas, fb.Dpi, visualBrushRenderer, canvas, surface, shim, fb);
         }
+
+        public IRenderTargetBitmapImpl CreateLayer(int pixelWidth, int pixelHeight)
+        {
+            // TODO: Get proper DPI here.
+            return new BitmapImpl(pixelWidth, pixelHeight, new Vector(96, 96));
```

Comment:
There is no way to obtain DPI at that point, since `ILockedFramebuffer` instance is not yet obtained. DPI is known in DrawingContext implementation.

Not sure how to deal with that.